wiremock/mappings/062-dps-frontend-assets-mapping.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
We need this mapping (over and above the actual files in `__files`) to add the cross origin header